plugins/database/postgresql/postgresql_test.go

```diff
@@ -155,6 +159,22 @@ func TestPostgreSQL_CreateUser(t *testing.T) {
 				GRANT SELECT ON ALL TABLES IN SCHEMA public TO "{{username}}";
 				GRANT SELECT ON ALL SEQUENCES IN SCHEMA public TO "{{username}}";`,
 			},
+			shouldTestCredsExist: true,
+		},
+		"reproduce https://github.com/hashicorp/vault/issues/6098": {
```

Comment:
Small nit, this might look odd in test results, maybe something like `"reproduce GH-6098"` along with a commented link?

Reply:
Updated in https://github.com/hashicorp/vault/pull/8512/commits/ad56d0fae9c77c5c00907c9f224907e84f1f865a